### PR TITLE
Add Kebechet App table

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Added new table and alembic version update.

### CODE REVIEW

**Comment by saisankargochhayat** [comment]
retest

File: thoth/storages/graph/models.py [hunk 1]
```diff
@@ -1510,6 +1530,7 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
         RPMRequirement,
         SoftwareEnvironment,
         VersionedSymbol,
+        KebechetGithubAppInstalltions
```
Comment (saisankargochhayat) [inline thoth/storages/graph/models.py:1533]: @pacospace Should I add it here?

Comment (pacospace) [reply thoth/storages/graph/models.py:1533]: yes it is a main model, but the list is in alphabetical order :)

Comment (saisankargochhayat) [reply thoth/storages/graph/models.py]: Thanks!

**Comment by saisankargochhayat** [comment]
recheck

File: thoth/storages/graph/models.py [hunk 2]
```diff
@@ -1471,6 +1471,24 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
     python_packages_metadata = relationship("HasMetadataDistutils", back_populates="python_package_metadata_distutils")
 
 
+class KebechetGithubAppInstalltions(Base, BaseExtension):
+    """Kebechet Github App Installations.
+
+    slug = namespace/repository (ex - thoth-station/advisor)
+    repo_name = repository (ex - advisor)
+    private = True or False
+    installation_id = proviced by github (ex - 236821515)
```
Comment (fridex) [inline thoth/storages/graph/models.py:1480]: provi**c**ed

File: thoth/storages/graph/models.py [hunk 3]
```diff
@@ -1471,6 +1471,24 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
...
+    """
+
+    __tablename__ = "kebechet_github_installations"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    slug = Column(Text, nullable=False, unique=True)
+    repo_name = Column(Text, nullable=True)
+    private = Column(Boolean, nullable=True)
```
Comment (fridex) [inline thoth/storages/graph/models.py:1488]: Not nullable here so we have 2 distinct values (private / not private).

BTW do we need to have nullables on other properties (repo name, installation id, ...).

Comment (saisankargochhayat) [reply thoth/storages/graph/models.py:1488]: We don't need them to be nullable, let's make nullable false on all of these. :)

File: thoth/storages/graph/models.py [hunk 4]
```diff
@@ -1485,6 +1503,7 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
         ExternalSoftwareEnvironment,
         HardwareInformation,
         InspectionRun,
+        KebechetGithubAppInstalltions,
```
Comment (pacospace) [inline thoth/storages/graph/models.py:1506]: typo here

File: thoth/storages/graph/models.py [hunk 5]
```diff
@@ -1471,6 +1471,24 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
     python_packages_metadata = relationship("HasMetadataDistutils", back_populates="python_package_metadata_distutils")
 
 
+class KebechetGithubAppInstalltions(Base, BaseExtension):
```
Comment (pacospace) [inline thoth/storages/graph/models.py:1474]: typo also here :)

File: thoth/storages/graph/models.py (hunk 4 above)
Comment (saisankargochhayat) [reply thoth/storages/graph/models.py]: Thanks :)

File: thoth/storages/graph/models.py (hunk 5 above)
Comment (saisankargochhayat) [reply thoth/storages/graph/models.py]: Sorry what's the typo in the docstring?

**Comment by saisankargochhayat** [comment]
> LGMT +1
> 
> Can be migrations squashed into one?

Thanks, I have changed it to one migration.

**Comment by saisankargochhayat** [comment]
recheck

**Pull request merged**